airbyte-webapp/src/components/GeographyDropdown/GeographyDropdown.tsx

```diff
@@ -0,0 +1,108 @@
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React, { useState } from "react";
+import { useIntl } from "react-intl";
+import { components, OptionProps, MenuListProps, StylesConfig } from "react-select";
+
+import "/node_modules/flag-icons/css/flag-icons.min.css";
+
+import { DropDown } from "components/ui/DropDown";
+import { DropdownProps } from "components/ui/DropDown";
+
+import { Geography } from "core/request/AirbyteClient";
+import { theme } from "packages/cloud/theme";
+
+import styles from "./GeographyDropdown.module.scss";
+
+const GeographyOption: React.FC<OptionProps> = (props) => {
+  const { formatMessage } = useIntl();
+  return (
+    <components.Option className={styles.option} {...props}>
+      <div className={styles.flag}>
+        <span className={`fi fi-${props.label === "auto" ? "us" : props.label}`} />
+      </div>
+      <span className={styles.label}>{formatMessage({ id: `geography.${props.label}` })}</span>
+    </components.Option>
+  );
+};
+
+const MenuList = (props: MenuListProps) => {
+  const { formatMessage } = useIntl();
+  return (
+    <components.MenuList {...props}>
+      {props.children}
+      <div className={styles.menuList}>
+        <FontAwesomeIcon icon={faPlus} />
+        <span>{formatMessage({ id: "geography.new" })}</span>
+      </div>
+    </components.MenuList>
+  );
+};
+
+interface GeographySelectOption {
+  label: Geography;
+  value: Geography;
+}
+
+export const GeographyDropdown: React.FC<DropdownProps<GeographySelectOption>> = ({ options }) => {
+  const [option, setOption] = useState();
+  const { formatMessage } = useIntl();
+  const handleOptionSelect = (e: any) => {
```

Comment:
We should not type something as `any` if not absolutely necessary. I don't see why we'd need to use `any` in this place, since we know that the element has a label: string, so we should rather type it as `{ label: string }`.

Reply:
@timroes got it! I knew this was not the right way to do it, but wanted to get it working. Next time I'll leave a comment/question!